main.py

```diff
@@ -95,17 +180,43 @@ def draw() -> Shape:
         if events.type == generate_new_round and current_round <= 24:
             current_round = current_round + 1
 
-            if current_round > 24:
+            if current_round > 44:
+                data_management.database_commit()
                 draw_end_game()
-            else:
+            if current_round < 12:
                 clickable = True
                 winning_shape = draw()
                 last_shape = winning_shape.type
+                continue
+            if 12 < current_round < 24:
```

Comment:
Might need you to explain this one as well, I mentioned the draw_second_stage method above

Reply:
Explained the second stage above, is that what you were asking?

Reply:
@pedrorribeiro Yeah, at the moment second_stage does the exact same thing as draw(), it doesn't add a delay on drawing the images to the screen which is where the confusion came from. We can continue discussion on how that works or will work in the above comment.